src/less/time-grid.less

```diff
@@ -94,6 +94,11 @@
     border-left-width: 0;
     border-right: 1px solid @cell-border;
   }
+
+  >.rbc-row.rbc-row-resource {
+      border-bottom: 1px solid @cell-border;
+      flex-shrink: 0;
```

Comment:
why is a new class needed? This change shouldn't break be changing any existing markup right?

Reply:
I added this in because otherwise, the new resource headers aren't given flexibility in terms of how they are rendered.

Using a custom header, this is what I see.
With the change:
<img width="1146" alt="screen shot 2019-01-22 at 8 48 20 am" src="https://user-images.githubusercontent.com/2019956/51543211-abb50780-1e22-11e9-948b-4fcf61491ae3.png">

Without the chance:
<img width="1129" alt="screen shot 2019-01-22 at 8 48 44 am" src="https://user-images.githubusercontent.com/2019956/51543317-e74fd180-1e22-11e9-85fe-e34aba8df7cd.png">

In fact, I think the I think the `> .rbc-row.rbc-row-resource {` that's under the `.rbc-time-header {` is not doing anything (as far as i could see anyway). But I left it there because I didn't want to break anything.